.gitignore

```
@@ -1,3 +1,9 @@
+data
+
+models
+
 venv
 
 .env
+
+*.pyc
```

```suggestion
data
models
venv

.env

*.pyc
```
`.gitignore` 파일에 개행이 많이 들어가 있어서 가독성이 떨어지는 것 같아요.
앞으로 확장성을 생각해서 [solaris33/pandas](https://github.com/solaris33/pandas/blob/master/.gitignore)처럼 관련있는 항목끼리 묶어서 정리하면 훨씬 읽기 쉬울 것 같습니다!

- 폴더 관련 (data/, models/, venv/)
- 환경 설정 (.env)
- Python 캐시 (*.pyc)
이런 식으로 묶어보면 어떨까요?

두 분의 의견이 궁금합니다!
@Eun0713 @devjaeyoon

---

좋습니다! `.gitignore`가 불필요하게 길어졌다고 코드를 작성하며 생각한 적 있었는데 짚어주셔서 감사드립니다!

---

말씀해주신 방식 좋은 것 같아요!
`.gitignore` 항목이 더 많아질 수 있다는 점을 고려하면, 관련된 항목끼리 묶어서 정리해두는 편이 가독성도 좋고 유지보수에도 유리할 것 같아요 :)